packages/react/src/ReactContext.js

```diff
@@ -78,18 +78,12 @@ export function createContext<T>(
     const consumer = {
       $$typeof: REACT_CONTEXT_TYPE,
       _context: context,
+      _calculateChangedBits: context._calculateChangedBits,
       Provider: context.Provider,
+      unstable_read: context.unstable_read,
```

Comment:
It's assigned right after, no? This is undefined.

Reply:
Moved the line above, my bad.